int32 marker_id
string marker
---
int32[] markers_id
string[] markers